src/components/NewComps/Testimonials/SpotlightTestimonials/SpotlightTestimonials.module.scss

```
@@ -0,0 +1,78 @@
+  }
+}
+
+.trackUp {
+  animation: scrollUp 36s linear infinite;
+}
+
+.trackDown {
+  animation: scrollDown 40s linear infinite;
+}
+
+@keyframes scrollUp {
```

Stylelint (17.14.1)

[error] Expected keyframe name "scrollUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)